src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -88,7 +88,7 @@ namespace cryptonote
 // advance which version they will stop working with
 // Don't go over 32767 for any of these
 #define CORE_RPC_VERSION_MAJOR 3
-#define CORE_RPC_VERSION_MINOR 8
+#define CORE_RPC_VERSION_MINOR 9
```

Comment:
This will conflict, we're already at 9 in master.

Reply:
OK, rebased and bumped the version to 10.